app/models/mentor/application.rb

```diff
@@ -0,0 +1,118 @@
+module Mentor
+  class Application
+    include ActiveModel::Model
+
+    DB = ActiveRecord::Base.connection
+
+    attr_accessor :id, :team_name
+    attr_accessor :project_id, :project_name
+    attr_accessor :project_plan, :why_selected_project
+    attr_accessor :student0, :student1
+    attr_accessor :first_choice
+
+    def choice
+      first_choice ? 1 : 2
+    end
+
+    def student0=(attrs)
+      attrs     = JSON.parse(attrs).instance_eval(&studentize)
```

Comment:
Makes sense! I'll come up with something. Just quickly add the "notes" before now.